utilities/serving_runtime.py

```diff
@@ -133,7 +133,7 @@ def update_model_dict(self) -> dict[str, Any]:
             _model_metadata.setdefault("annotations", {})["enable-route"] = "true"
 
         if self.enable_auth:
-            _model_metadata.setdefault("annotations", {})["enable-route"] = "true"
+            _model_metadata.setdefault("annotations", {})["enable-auth"] = "true"
```

Comment:
do we need to add 
annotations in constants.py?

Reply:
we can in another pr